docs/en/sql-reference/sql-statements/data-definition/ALTER_TABLE.md

```diff
@@ -291,43 +411,50 @@ ORDER BY (column_name1, column_name2, ...)
 
 Note:
 
-1. All columns in the index must be written.
-2. The value column is listed after the key column.
+- All columns in the index must be written.
+- The value column is listed after the key column.
 
-#### Add a generated column (from v3.1)
+#### Modify columns of the sort key in a Primary Key table
 
-Syntax:
+<!--Supported Versions-->
 
-```sql
-ALTER TABLE [<db_name>.]<tbl_name>
-ADD COLUMN col_name data_type [NULL] AS generation_expr [COMMENT 'string']
-```
+Syntax:
 
-You can add a generated column and specify its expression. [The generated column](../generated_columns.md) can be used to precompute and store the results of expressions, which significantly accelerates queries with the same complex expressions. Since v3.1, StarRocks supports generated columns.
+```SQL
+ALTER TABLE [<db_name>.]<table_name>
+[ order_desc ]
 
-#### Modify table properties
+order_desc ::=
+    ORDER BY <column_name> [, <column_name> ...]
+```
 
-Currently, StarRocks supports modifying the following table properties:
+Example:
 
-- `replication_num`
-- `default.replication_num`
-- `storage_cooldown_ttl`
-- `storage_cooldown_time`
-- Dynamic partitioning related properties
-- `enable_persistent_index`
-- `bloom_filter_columns`
-- `colocate_with`
+For example, the original table is a Primary Key table where the sort key is coupled with the primary key `dt, order_id`.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/cf5c21c2-7f2c-4b17-9193-7f8169494feb)
I think the Chinese means both the sort key and the primary key is composed of the dt and order_id columns. Pls check.